examples/cli/README.md

```diff
@@ -6,7 +6,8 @@ It can be used as a reference for using the `whisper.cpp` library in other proje
 ```
 ./build/bin/whisper-cli -h
 
-usage: ./build-pkg/bin/whisper-cli [options] file0.wav file1.wav ...
+usage: ./build-pkg/bin/whisper-cli [options] file0 file1 ...

```

Comment:
I think this might be incorrect in the current README.md and that the correct path should  be `./build/bin/whisper-cli` and not `./build-pkg`. For example, this is the output for me locally:
```console
$ ./build/bin/whisper-cli --help

usage: ./build/bin/whisper-cli [options] file0 file1 ...
supported audio formats: flac, mp3, ogg, wav
...
```

Reply:
Nice catch. Fixed